core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html

```diff
@@ -838,7 +867,207 @@ <h2>Suggestions to review</h2>
 
 {% block footer_js %}
   {{ super() }}
+  <script src="/templates/dev/head/mathjaxConfig.js"></script>
```

Comment:
Hi @nithusha21, do you remember if when you were adding this script-import if you copied from somewhere or if you added only script-imports that were really needed for this page?

Reply:
This page displays the state editor(for creation of questions). It was technically a copy over from the other questions editor.

Reply:
Sorry, I phrased my question wrong, I was talking about all the script-imports in this file because some of them seem unused.

Reply:
Oh I see, I am not sure, but I think it was a direct copy paste from the exploration editor or the question editor page. They might be unused (I don't recollect checking each one of them).

Reply:
Ok, thanks for the info.